js/fetch_test.ts

```diff
@@ -88,6 +88,15 @@ testPerm(
   }
 );
 
+testPerm({ net: true }, async function fetchWithRedirection(): Promise<void> {
+  const response = await fetch("http://localhost:4545/tests"); // will redirect to /tests/
```

Comment:
We have a special redirect server on localhost:4546 and a double redirect server on :4548

Check out https://github.com/denoland/deno/blob/988bcbb8842d12202f278808698a6b546718e764/tools/http_server.py#L139-L153

I think hitting the double redirect server would be a valuable unit test here.